api/src/db/migrations/v141_sync_migrated_and_newly_created_users.ts

```diff
@@ -0,0 +1,829 @@
+import { v140Business, v140UserData } from "@db/migrations/v140_modify_home_health_industry";
+import { randomInt } from "@shared/intHelpers";
+
+export const migrate_v140_to_v141 = (v140Data: v140UserData): v141UserData => {
```

Comment:
These migration specific updates are important to check in this PR to see if there are any errors in how we are bringing users up to current version of UserData.